apps/change-log.md

```diff
@@ -14,6 +14,28 @@ Of course, we will continue to inform you if there are any breaking changes to t
 
 <hr>
 
+## 2023-04-13
+
+### Changes with software release 7.82.0
+
+#### <i class="fa fa-plus"></i> New
```

Comment:
Why is this sometimes "fa" and sometimes "fas"?

Reply:
@ywolpers For "plus" it seems that both versions are working. I'll still change it to "fas". Thanks :)